.github/workflows/macos_build.yaml

```
@@ -47,41 +67,98 @@
-          test -d flites.app || (echo "App bundle not found" && exit 1)
-          # List app bundle contents
-          ls -la flites.app/Contents/MacOS/ 
+          # Check if the .app bundle directory was created by the build step
+          APP_BUNDLE="apps/flites/build/macos/Build/Products/Release/${{ env.APP_NAME_LOWER }}.app" # Use lowercase
+          echo "Checking for ${APP_BUNDLE}"
+          if [ -d "${APP_BUNDLE}" ]; then
+            echo "macOS build output (.app bundle) found."
+          else
+            echo "Error: macOS build output (.app bundle) not found!"
+            exit 1
+          fi
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)